Source/CesiumRuntime/Private/CesiumCreditSystem.cpp

```diff
@@ -163,16 +163,43 @@ void ACesiumCreditSystem::Tick(float DeltaTime) {
       creditsToShowThisFrame.size() != _lastCreditsCount ||
       _pCreditSystem->getCreditsToNoLongerShowThisFrame().size() > 0;
   if (CreditsUpdated) {
+    bool firstScreenCredit = false;
+    std::string onScreenCreditString =
+        "document.write('<!DOCTYPE html><head><base target=\"_blank\">"
+        "<style>body{position:fixed;bottom:0;color:white;font-size:10px;font-family:sans-serif;}"
+        "div{display:inline;}a{color:white}</style>"
+        "<meta charset=\"utf-8\"/></head><body>";
     std::string creditString =
         "<head>\n<meta charset=\"utf-16\"/>\n</head>\n<body style=\"color:white\"><ul>";
+    bool hasPopupCredits = false;
     for (size_t i = 0; i < creditsToShowThisFrame.size(); ++i) {
-      creditString +=
-          "<li>" + _pCreditSystem->getHtml(creditsToShowThisFrame[i]) + "</li>";
+      if (_pCreditSystem->shouldBeShownOnScreen(creditsToShowThisFrame[i])) {
+        if (!firstScreenCredit) {
+          firstScreenCredit = true;
+        } else {
+          onScreenCreditString += "<span> &bull; </span>";
+        }
+        onScreenCreditString +=
+            "<div>" + _pCreditSystem->getHtml(creditsToShowThisFrame[i]) +
+            "</div>";
+      } else {
+        creditString += "<li>" +
+                        _pCreditSystem->getHtml(creditsToShowThisFrame[i]) +
+                        "</li>";
+        hasPopupCredits = true;
+      }
     }
     creditString += "</ul></body>";
     Credits = UTF8_TO_TCHAR(creditString.c_str());
-
+    if (hasPopupCredits) {
+      onScreenCreditString +=
+          "<span> </span><a href=\"https://data-attribution-popup\">Data attribution</a>";
```

Comment:
I'd suggest using a made-up protocol to make it clear this is not a real web URL. `cesium-unreal://data-attribution-popup` maybe. Unless that doesn't work for some reason that isn't obvious to me? Either way, a comment explaining the weird URL would be helpful.

Reply:
> In your screenshot, the Bing logo is at the end, which doesn't seem right. Is that because credits on raster overlays aren't considered to be in every tile the way tileset-level credits now are?

I thought that was the correct location because thats the same way CesiumJS does it. The Bing logo is only added once per frame. https://github.com/CesiumGS/cesium/blob/7e70b89cd1a3ced4c57cf8ccdf05f24585043cbc/Source/Scene/GlobeSurfaceTileProvider.js#L361

I tried to imitate CesiumJS as much as possible so the credits display have almost the same behavior.